electron/src/paths.ts

```diff
@@ -10,24 +10,51 @@ const DEV_USER_NUMBER = process.env.ACORN_AGENT_NUM
 
 const MIGRATION_FILE_NAME_PREFIX = `data-migration-for-integrity-version-`
 
-const USER_DATA_PATH = app.isPackaged
+const DEV_USER_DATA_PATH = app.isPackaged
   ? app.getPath('userData')
   : path.join(__dirname, `../../user-data/agent${DEV_USER_NUMBER}`)
 
 const USER_DATA_MIGRATION_FILE_PATH = path.join(
-  USER_DATA_PATH,
+  DEV_USER_DATA_PATH,
   `${MIGRATION_FILE_NAME_PREFIX}${INTEGRITY_VERSION_NUMBER}`
 )
 const PREV_VER_USER_DATA_MIGRATION_FILE_PATHS = [
   // Acorn 3
-  path.join(USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}5`),
+  path.join(DEV_USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}5`),
   // Acorn 4
-  path.join(USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}6`),
+  path.join(DEV_USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}6`),
   // Acorn 5
-  path.join(USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}8`),
+  path.join(DEV_USER_DATA_PATH, `${MIGRATION_FILE_NAME_PREFIX}8`),
 ]
+
+const PROJECTS_HAPP_PATH = app.isPackaged
+  ? path.join(app.getAppPath(), '../app.asar.unpacked/binaries/projects.happ')
+  : path.join(app.getAppPath(), 'binaries/projects.happ')
+
+const PROFILES_HAPP_PATH = app.isPackaged
+  ? path.join(app.getAppPath(), '../app.asar.unpacked/binaries/profiles.happ')
+  : path.join(app.getAppPath(), 'binaries/profiles.happ')
+
+const DATASTORE_PATH = app.isPackaged
```

Comment:
This whole thing can simply become path.join(USER_DATA_PATH, databases …etc

Since the other var already alters per environment